embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.java

```diff
@@ -1538,7 +1538,7 @@ FlutterInternalInterface getFlutterInternalInterface() {
 
     private void onActivityReported() {
```

Comment:
I would LOVE to get rid of this. I'm sure we have calls that are missing this right now.

That said, I like the idea of someone being able to notify the caching mechanism that something has changed and the payload needs to be regenerated again. Perhaps this is something we can work on when the orchestration and new payload population work is done so we do a better job for background activities and also be more efficient rather than always saving every 2s for foreground sessions.

Reply:
Yep, it would be good to get rid of this. There's a task to align the session/BA persistence with each other, so we'll see whether it's possible as part of that.